modules/ppcp-googlepay/resources/js/GooglepayButton.js

```diff
@@ -515,7 +515,9 @@ class GooglepayButton extends PaymentButton {
 					)
 				) {
 					paymentDataRequestUpdate.newShippingOptionParameters =
-						updatedData.shipping_options;
```

Comment:
The `shipping_options` list contains the non-standard property `cost` which is used by this component to include shipping cost in the total price.

This commit strips any non-standard properties to prevent errors